apps/coscrad-frontend/src/components/alphabet/AlphabetPage.tsx

```diff
@@ -15,17 +15,31 @@ export const AlphabetPage = (): JSX.Element => {
 
     const endpoint = `${baseUrl}/games/${alphabetConfig.alphabetChartName}`;
 
-    const baseMediaUrl = `${baseUrl}/resources/mediaItems/download?name=`;
+    const baseMediaUrl = `${alphabetConfig.baseDigitalAssetUrl}`;
 
     const [alphabet, setAlphabet] = useState<IAlphabetChart | null>(null);
+    const [isLoading, setIsLoading] = useState(true);
```

Comment:
We might want to refactor to use the `Loadable` abstraction to take care of the loading state logic for us. Let's add a story for this.